api/libs/scenarios-planning-unit/src/domain/lock-planning-units.ts

```diff
@@ -0,0 +1,11 @@
+import { differenceWith } from 'lodash';
+
+export const lockPlanningUnits = (
+  planningUnitsIds: string[],
+  availablePlanningUnitsIds: string[],
+): { errors: string[] } => {
+  const diff = differenceWith(planningUnitsIds, availablePlanningUnitsIds);
```

Comment:
`_.difference()` should be enough here as we don't pass in a comparator function

Reply:
@hotzevzl fixed in https://github.com/Vizzuality/marxan-cloud/pull/224/commits/8b9c4a7bbed5d99bc818bedd331199dc29415f8e